lua/lazyvim/plugins/extras/formatting/biome.lua

```diff
@@ -0,0 +1,61 @@
+---@diagnostic disable: inject-field
+if lazyvim_docs then
+  -- Enable this option to avoid conflicts with Prettier.
+  vim.g.lazyvim_prettier_needs_config = true
+end
+
+-- https://biomejs.dev/internals/language-support/
+local supported = {
+  "astro",
+  "css",
+  "graphql",
+  -- "html",
+  "javascript",
+  "javascriptreact",
+  "json",
+  "jsonc",
+  -- "markdown",
+  "svelte",
+  "typescript",
+  "typescriptreact",
+  "vue",
+  -- "yaml",
+}
+
+return {
+  {
+    "williamboman/mason.nvim",
+    opts = { ensure_installed = { "biome" } },
+  },
+
+  {
+    "stevearc/conform.nvim",
+    optional = true,
+    ---@param opts ConformOpts
+    opts = function(_, opts)
+      opts.formatters_by_ft = opts.formatters_by_ft or {}
+      for _, ft in ipairs(supported) do
+        opts.formatters_by_ft[ft] = opts.formatters_by_ft[ft] or {}
+        table.insert(opts.formatters_by_ft[ft], "biome")
+      end
+
+      opts.formatters = opts.formatters or {}
+      opts.formatters.biome = {
+        condition = function(_, ctx)
+          return vim.fs.find({ "biome.json", "biome.jsonc" }, { path = ctx.filename, upward = true })[1]
+        end,
+      }
```

Comment:
You could also use this: https://github.com/stevearc/conform.nvim/blob/6a28c90082a67f62a323ab90d988e000a718a8c7/lua/conform/formatters/biome.lua#L11-L14

And just set:

```lua
opts.formatters.biome = {
  require_cwd = true,
}
```